Wikipedia/Code/UIImageView+WMFContentOffset.m

```diff
@@ -38,7 +38,7 @@ - (void)wmf_resetContentsRect {
 }
 
 - (void)wmf_topAlignContentsRect:(UIImage *)image {
-    self.layer.contentsRect = CGRectMake(0, 0, 1, (image.size.width / image.size.height) / 2.f);
+    self.layer.contentsRect = CGRectMake(0, 0, 1, MIN(image.size.width / image.size.height / 2.f, 1.0f));
```

Comment:
I meant to mention this before - but i'm not sure if the f suffix is still required - with 64 bit phones aren't they all doubles anyway?